### PR TITLE
Add CORS requirement for custom fonts

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md [hunk 1]
```diff
@@ -36,12 +36,19 @@ For **Basic Styling**, you can set your default email and content background col
...
+- The file URL must point to a .css file (not a .woff, .otf, etc.).
+- The custom font name must match the name of the font face in the .css file.
+
 To add a custom font:
 
 1. Click **Add a custom font** below the **Default Font Name** of the styling section.
-2. Enter the font's name and source file URL. This source file URL must point to a style sheet like a CSS file. For the **Font Name** field, enter the same font name as your custom font source file. Ensure that the name is capitalized and spaced correctly. Enter the corresponding **Font URL**. 
-3. Check that the preview shows your custom font before saving. 
-4. Click **Save** to use the custom font as your default email font. 
+2. For the **Font Name** field, enter the same font name as your custom font source file. Ensure that the name is capitalized and spaced correctly.
```
Comment (djhanggi) [inline _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md:48]:
```suggestion
2. For the **Font Name** field, enter the same font name that appears in your custom font source file. Ensure that the name is capitalized and spaced correctly. 
```

File: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md [hunk 2]
```diff
@@ -36,12 +36,19 @@ For **Basic Styling**, you can set your default email and content background col
 
 With custom fonts, you can manually add a web font for branding consistency across various email platforms. You can add one custom font per section. 
 
+Before adding a custom font, check that the font meet the following requirements:
+
+- CORS must be enabled on the server that provides the custom font. This is usually handled by your IT team.
```
Comment (djhanggi) [inline _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md:41]: Should we link to Mozilla docs? https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
```suggestion
- CORS must be enabled on the server that provides the custom font file. This is usually handled by your IT team.
```

**Comment by djhanggi** [comment]
I think this is good!

File: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md [hunk 3]
```diff
@@ -36,12 +36,19 @@ For **Basic Styling**, you can set your default email and content background col
 
 With custom fonts, you can manually add a web font for branding consistency across various email platforms. You can add one custom font per section. 
 
+Before adding a custom font, check that the font meet the following requirements:
+
+- CORS must be enabled on the server that provides the custom font file. This is usually handled by your IT team.
```
Comment (djhanggi) [inline _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md:41]:
```suggestion
- CORS must be enabled on the server that provides the custom font file. This is usually handled by your IT team.
  - The file must have the header: `Access-Control-Allow-Origin: *`
```

Something like this, though this is pretty technical, so relying on your double check!

File: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md [hunk 4]
```diff
@@ -36,12 +36,20 @@ For **Basic Styling**, you can set your default email and content background col
 
 With custom fonts, you can manually add a web font for branding consistency across various email platforms. You can add one custom font per section. 
 
+Before adding a custom font, check that the custom font file meets the following requirements:
+
+- CORS must be enabled on the server that provides the custom font file. This is usually managed by your IT team. 
+  - The custom font file must have the header: `Access-Control-Allow-Origin: *`
```
Comment (djhanggi) [inline _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md:42]: should this have a period?

**Review by djhanggi — APPROVED** [review]

File: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md [hunk 5]
```diff
@@ -36,12 +36,20 @@ For **Basic Styling**, you can set your default email and content background col
...
+- The file URL must point to a CSS file (not WOFF, OTF, etc.).
+- The custom font name must match the name of the font face in the CSS file.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md:44]:
```suggestion
- CORS must be enabled on the server that provides the custom font file. This is usually managed by your IT team. 
  - The custom font file must have the header: `Access-Control-Allow-Origin: *`
- The file URL must point to a CSS file (not WOFF, OTF, etc.)
- The custom font name must match the name of the font face in the CSS file
```

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**